bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -188,12 +188,20 @@ spec:
           # As the secret mount is optional we must wait for the files to be present.
           # The service is created in monitor.yaml and this is created in sdn.yaml.
           # If it isn't created there is probably an issue so we want to crashloop.
-          TS=$(curl \
-            --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
-            -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-            "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/namespaces/openshift-sdn/services/sdn" |
-              python -c 'import json,sys; print(json.load(sys.stdin)["metadata"]["creationTimestamp"])'
-          )
+          retries=0
+          until [ "$retries" -ge 20 ]
+          do
+            TS=$(curl -s \
+              --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+              -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
+              "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/namespaces/openshift-sdn/services/sdn" |
+                python -c 'import json,sys; print(json.load(sys.stdin)["metadata"]["creationTimestamp"])' &&
+                  break ||
+                  echo "WARN: Failed to get sdn service from API" 1>&2
```

Comment:
If this is _expected_ it should be a "warning". And you should say you're retrying.

Also, can you use `if`/`else` rather than `&&`/`||`? (I realize the variable assignment might make that tricky, so maybe no...)